x-pack/plugins/security_solution/public/common/components/events_viewer/events_viewer.test.tsx

```diff
@@ -189,19 +189,14 @@ describe('EventsViewer', () => {
         wrapper.find(`[data-test-subj="expand-event"]`).first().simulate('click');
       });
 
-      waitFor(() => {
-        expect(mockDispatch).toBeCalledTimes(2);
+      await waitFor(() => {
```

Comment:
This test was never running because of the missing `await`. I updated the assertion to pass.